packages/auth/src/signature/middleware.test.ts

```diff
@@ -32,7 +32,8 @@ import {
 
 describe('Signature Service', (): void => {
   let deps: IocContract<AppServices>
-  let appContainer: TestContainer
+  const appContainers: TestContainer[] = []
```

Comment:
Rather than a single appContainer, this allows for multiple with different configs. Because the container takes a config as input and creates a singleton this is required if configs are materially different per test.